app/controllers/moderator/post/approvals_controller.rb

```diff
@@ -6,7 +6,7 @@ class ApprovalsController < ApplicationController
       respond_to :json, :xml, :js
 
       def create
-        cookies.permanent[:moderated] = Time.now.to_i
+        #cookies.permanent[:moderated] = Time.now.to_i
```

Comment:
Instead of commenting these lines, you should just delete them outright. (That is the point of version control, after all.)